openfisca_web_site/templates/france/fr/tree/demo.mako

```diff
@@ -116,7 +116,7 @@ from openfisca_web_site import urls
 
     function callOpenFiscaAPI(callback) {
       var xmlhttp = new XMLHttpRequest();
-      var url = "https://api-test.openfisca.fr/calculate";
+      var url = "https://fr.openfisca.org/api/v18/calculate";
```

Comment:
The last version is 20 but I guess version 18 is OK for now as: 
- mesaides is still using version 19
- the standard installation on windows lead to a version 18